src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Index.txt

```diff
@@ -0,0 +1,20 @@
+>> Index([1, 2, 3, 4, 5], 3)
+{Value:3}
```

Comment:
The excel function ([https://support.microsoft.com/en-us/office/index-function-a5dcf0dd-996d-40a4-a822-b56b061328bd](https://support.microsoft.com/en-us/office/index-function-a5dcf0dd-996d-40a4-a822-b56b061328bd))  returns  a scalar. 

Should this just return 3, not {value:3} ?

Reply:
`[1,2,3]` is just syntactic sugar for `Table({Value: 1}, {Value: 2}, {Value: 3})`. We should return a Record and not have special cases for single-column tables.

Reply:
Both statements are true, so there's a contradiction to resolve: excel consistency vs. sct semantics ... can we confirm with Greg?

Reply:
Discussed offline ... this is also consistent with `First(x)` = `Index(x,1)`. 
So this is probably inevitable. But we should not it's different than excel.